.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Should we install `prettier` (I don't think it's been installed as a `devDependency`)?

Reply:
Yes! :) Thanks